dds/DCPS/DiscoveryBase.h

```diff
@@ -86,24 +87,70 @@ namespace OpenDDS {
     }
 
     struct DcpsUpcalls : ACE_Task_Base {
+      inline bool has_timeout() {
```

Comment:
```suggestion
      bool has_timeout() const {
```
already inline (defined within class definition)